src/progressbar/progressbar.ts

```diff
@@ -0,0 +1,50 @@
+import {Component, Input, ElementRef, Renderer} from 'angular2/angular2';
+
+@Component({
+  selector: 'ngb-progressbar',
+  template: `
+    <progress class="progress {{class}} {{type && 'progress-' + type}}" [value]="value" [max]="max">
+      <div class="progress">
+        <span class="progress-bar" [style.width.%]="calculatePorcentage()"><ng-content></ng-content></span>
+      </div>
+    </progress>
+  `
+})
+export class NgbProgressbar {
+  _value: number;
+  _class: string;
+  min = 0;
+
+  @Input() set class
+  (newClasses: string) {
+    this._class = newClasses;
+    const initialClasses = newClasses && newClasses.split(' ');
+    initialClasses.forEach(className => this.toggleClass(className));
```

Comment:
Bah, I really don't see why we need it... If someone removed classes from host, they are gone already, no?

If you want to copy classes from host to a progress element just do this, as an entire string. Having said this I _really_ don't think we should be basing our API css classes. IMO we should have our own attributes and map those to the corresponding bootstrap css